docs/contribution/community-components.md

```diff
@@ -0,0 +1,299 @@
+# Community components
+
+### What are community components?
+
+Community Components is our new library of custom components contributed by you.
```

Comment:
WH has a a group of Carbon components that are built in Handlebars that are exportable and re-useable... would that live in the community components under a single package for "handlebars"? or would that be something that would still want to be in the core library since they are duplicate components from what is in core?